code/src/index.css

```diff
@@ -1,13 +1,84 @@
+* {
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+}
+
+html {
+	width: 100vw;
+	height: 100vh;
+}
+
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  font-family: 'Roboto', sans-serif;
+  background-image: url('./images/pizza-flip.jpg');
+  background-size: cover;
+  /* background-repeat: no-repeat; */
```

Comment:
What happens if you comment this in and put the background-image in the container instead? I had the same issue with the image not covering the screen even though I had background-size: contain; and for me it solved it by moving the image to my section/container instead